PendingReleaseNotes

```diff
@@ -1,324 +1,8 @@
->=15.0.0
+>=16.0.0
 --------
 
 * CVE-2020-10736: Fixes an authorization bypass in monitor and manager daemons
```

Comment:
this is in 15.2.2

Reply:
yeah, I guess CVEs fixed before 16.2.0 need not be mentioned in the Pacific release notes.